app/api-v1/routes/order.js

```diff
@@ -1,11 +1,16 @@
 // eslint-disable-next-line no-unused-vars
+const { BadRequestError } = require('../../utils/errors')
 module.exports = function (orderService) {
   const doc = {
     GET: async function (req, res) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      const { statusCode, result } = await orderService.postOrder(req.body)
+      if (statusCode != 201) {
+        throw new BadRequestError({ message: '400 Bad Request', service: 'order' })
+      }
```

Comment:
the whole if statement could be removed? At the moment, `postOrder` only returns a `201` or throws an error already